tools/wt_verify/wt_verify.py

```diff
@@ -355,21 +571,41 @@ def main():
         print(str(e), file=sys.stderr)
         sys.exit(1)
 
-    parsed_data = parse_output()
+    parsed_data = None
+
+    if "dump_pages" in command:
+        parsed_data = parse_dump_pages()
+    elif "dump_blocks" in command:
+        parsed_data = parse_dump_blocks()
+    else:
+        assert False, f"Unexpected command '{command}'"
```

Comment:
Doesn't need to be fixed in this PR, but in general it's more Pythonic to throw exceptions - any caller has a chance to catch them, and the error messages are cleaner.

Reply:
https://jira.mongodb.org/browse/WT-13027